src/test/java/nl/tudelft/ti2206/group9/level/StateTest.java

```diff
@@ -79,5 +79,18 @@ public void testSetCoins() {
 		State.setCoins(1);
 		assertEquals(1, State.getCoins());
 	}
+	
+	@Test
+	public void testGetHighScore() {
```

Comment:
This method does not only test the functionality of getHighscore(). Naming should be changed.

< naming, review, testing